orchestra/tests/helpers/fixtures.py

```diff
@@ -245,4 +251,52 @@ def setup_models(test_case):
                 task=task,
                 status=assignment_status,
                 assignment_counter=i,
-                in_progress_task_data=task_data)
+                in_progress_task_data=task_data,
+                start_datetime=parse(
+                    '2015-10-12T0{}:00:00+00:00'.format(2 + i)))
+
+
+def task_back_and_forth(test):
+    task = test.tasks['rejected_review']
+
+    test._submit_assignment(
+        test.clients[6], task.id, seconds=35)
+    test._submit_assignment(
+        test.clients[7], task.id, command='reject', seconds=36)
+    test._submit_assignment(
+        test.clients[6], task.id, seconds=37)
+    test._submit_assignment(
+        test.clients[7], task.id, command='accept', seconds=38)
+
+    # Fill out the snapshots for all assignments
+    assignments = assignment_history(task)
+    first_assignment = assignments[0]
+    second_assignment = assignments[1]
+    third_assignment = assignments[2]
+
+    first_assignment.snapshots['snapshots'] = deepcopy(
+        second_assignment.snapshots['snapshots'])
+    second_assignment.snapshots['snapshots'] = (
+        deepcopy(third_assignment.snapshots['snapshots']) +
+        second_assignment.snapshots['snapshots'])[:-1]
+
+    def fix_datetimes(snapshots, new_datetimes):
+        for snapshot, new_datetime in zip(snapshots, new_datetimes):
+            snapshot['datetime'] = new_datetime
+
+    fix_datetimes(
+        first_assignment.snapshots['snapshots'],
+        ['2015-10-12T02:02:00+00:00', '2015-10-12T03:05:00+00:00'])
+    fix_datetimes(
+        second_assignment.snapshots['snapshots'],
+        ['2015-10-12T03:01:00+00:00', '2015-10-12T03:07:00+00:00',
+         '2015-10-12T04:03:00+00:00', '2015-10-12T04:10:00+00:00'])
+    fix_datetimes(
+        third_assignment.snapshots['snapshots'],
+        ['2015-10-12T04:02:00+00:00', '2015-10-12T04:13:00+00:00'])
+
```

Comment:
otherwise the timestamp is datetime.now, which we can't test against.  these times are predictable distance apart, so I can test the resulting dataframe.  worth a comment?